### PR TITLE
 Remove redundant ballerina commands and content in ballerina.io

### PR DESCRIPTION
**Pull request by NatashaWso2**
- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md [hunk 1]
```diff
@@ -125,20 +116,27 @@ service<network:Service> hello bind { port:9090 } {
 ```
 
 ### Package Version Dependency
-If your source file is not part of a project, then you can explicitly manage version dependencies of imported packages within the source file using the `import` statement.
+In the future we will be extending the `import` statement to explicitly manage version dependencies of imported packages within the source file if it is not a part of a project.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Not sure if we should be saying what we're going to implement in future in this doc. The scope should be on how existing version dependencies happen IMO.

File: website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md [hunk 2]
```diff
@@ -264,41 +259,30 @@ A repository is a collection of packages. A repository helps organize packages u
...
-If a package is discovered at Ballerina Central, the build system will download the package's files before installing into both the home and project repository for reuse. 
+If a package is discovered at Ballerina Central, the build system will download the package's files before installing into the home repository for reuse. 
 
 ### Package Installation
 When building a package in a project, that package is automatically installed into the project's local repository. That package can be shared across other projects by installing it into the home repository.
 
-To install all packages in a project:
+The org-name and the version of the package will be read from the manifest file `Ballerina.toml` inside the project.
+
+We will extend the `push` command in the future to allow users to push packages into any remote registry hosted at a domain referenced by a URL, such as a private registry used by corporations.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Same comment as earlier.

File: website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md [hunk 3]
```diff
@@ -315,32 +299,23 @@ Remotely hosted repositories, such as Ballerina Central, can each have their own
...
-ballerina pull <org-name>/<package-name>[:<version>] [--repository=<url>]
+ballerina pull <org-name>/<package-name>[:<version>]
 ```
 
-Projects that perform dependency analysis will automatically pull packages into the home repository and also copy them into the project repository.
+Projects that perform dependency analysis will automatically pull packages into the home repository.
 
 ### Pushing Packages Into Remote Repositories
-"Pushing" a package uploads the associated package files and installs the package into a remote repository. If you do not specify a remote repository URL, then Ballerina Central is assumed. 
+"Pushing" a package uploads the associated package files and installs the package into a remote repository which is Ballerina Central.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Add a comma before 'which'.

Comment (NatashaWso2) [reply website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Fixed with 59edbb9

File: website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md (hunk 2 above)
Comment (NatashaWso2) [reply website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Remove the line with 59edbb9

File: website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md (hunk 1 above)
Comment (NatashaWso2) [reply website/two-column-pages/docs/learn/how-to-structure-ballerina-code.md]: Remove the line with 59edbb9

**Pull request merged**